doc/guides/spatial/guide_spatially_structured_networks.rst

```diff
@@ -1492,16 +1492,60 @@ Adding masks in a module
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 If using parameters to define a connection boundary is not efficient enough, or
-if you need more flexibility in defining the mask, you can add a custom mask,
-written in C++, and add it to NEST via an extension module. For more information
-on writing such modules, see the
-`NEST extension module repository <https://github.com/nest/nest-extension_module>`_.
+if you need more flexibility in defining the mask, you have to add a custom
```

Comment:
I think this section has [been removed from master!](https://github.com/nest/nest-simulator/commit/83d64741f03d8ac5f39511fc141762ad35dd7810#diff-a530e935653b38da12a64ec23693f874), MyModule is now factored out into a separate repository, and is no longer part of `examples/MyModule`. I therefore think you can remove most of it.